platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigratableTest.java

```diff
@@ -505,6 +523,11 @@ private Configuration getConfigurationForExportSystem(String pid) {
     }
     props.put("service.pid", pid);
     props.put("schema", "http://www.opengis.net/cat/csw/2.0.2");
+    if (pid.equals(METACARD_VALIDITY_FILTER_PLUGIN_PID)) {
+      props.put("attributeMap", new String[] {"invalid-state=data-manager", "system-user"});
```

Comment:
✏️ Technically, these 2 String elements should be a single String element. If you were to put a second element in this array it would look like this
`new String[] {"invalid-state=data-manager,system-user", "test-claim=test-role1,test-role2"}`
and the output that you assert on on line 427 would be 
`"invalid-state=localhost-data-manager,system-user,test-claim=test-role1,test-role2"`